tasks/main.yml

```diff
@@ -1,2 +1,24 @@
 ---
-# tasks file for skeleton
+- name: Check if Metasploit Framework is already installed
+  ansible.builtin.stat:
+    path: "{{ install_directory }}"
+  register: msf_directory
+
+- name: Install Metasploit Framework
+  block:
+    - name: Download Metasploit Framework Installer
+      ansible.builtin.get_url:
+        url: https://raw.githubusercontent.com/rapid7/metasploit-omnibus/master/config/templates/metasploit-framework-wrappers/msfupdate.erb
```

Comment:
Given that we run this shell script blindly I think we should pin to the latest commit for the file. I'm also open to pinning to the current tag if tying it to a commit seems unwieldy. The commit or tag could also be a variable for the role with a default so it's easy to override as needed. I am mostly just against blindly pulling down whatever has been merged into the `master` branch. Thoughts @cisagov/team-ois?

```suggestion
        url: https://raw.githubusercontent.com/rapid7/metasploit-omnibus/68a9de2b15dea8f3c7b1cf841874dee7ca4754f5/config/templates/metasploit-framework-wrappers/msfupdate.erb
```

Reply:
I'm on board with pinning to a version and having a default.  I don't have a strong preference for pinning to a commit vs. a tag.

Reply:
We don't do this elsewhere, and we were advised by the former tech council that the assessors need the latest and greatest.  To my knowledge this order has not been overridden or rescinded.

Reply:
This is not the version of the Metasploit Framework that is installed. This is the specific version of the installer *only*. The installer sets up the package manager and installs whatever is found there except for macOS which downloads the latest package available directly from the Metasploit website..

Reply:
OK, I'm fine with pinning and having a default.  We shouldn't do it via your commit, though, but via a role `default`.

Reply:
I am making the change now.